.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- 
+---------------------------------------------------
+GENERAL SUPPORT INFORMATION
+---------------------------------------------------
+
+The GitHub issue tracker is for bug reports and feature requests.
+General support can be found at the following locations:
+
+Example configs:
+
+3.0 - https://github.com/inspircd/inspircd/tree/master/docs/conf
+2.0 - https://github.com/inspircd/inspircd/tree/insp20/docs/conf
+
+Wiki:
+
+https://wiki.inspircd.org/
+
+IRC:
+
+irc.inspircd.org #inspircd
+
+
+-->
+
+**Description**
+
+<!--
+Briefly describe the problem you are having in a few paragraphs.
+-->
+
+**Steps to reproduce the issue:**
+1.
+2.
+3.
+
+**Describe the results you received:**
+
+
+**Describe the results you expected:**
+
+
+**Additional information you deem important (e.g. issue happens only occasionally):**
+
+**Output of `./inspircd version`:**
```

Comment:
The helper script is not shipped on every platform. You might want to mention that. On platforms without the helper script `inspircd --version` is used.